packages/framework/synthesize/src/dependencyContainer.ts

```diff
@@ -103,7 +107,9 @@ export class DependencyContainer<TMap> implements IFluidDependencySynthesizer {
 					return parent.getProvider(provider);
 				} else {
 					// older implementations of the IFluidDependencySynthesizer exposed getProvider
+					// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 					const maybeGetProvider: { getProvider?(provider: string & keyof TMap) } =
+						// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
We're assigning to a specific type here. Let's cast to that instead of `any`

Reply:
Okok